packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/14-runtime-layout-switching.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: runtime-layout-switching
+title: Runtime layout switching
+---
+
+## Introduction
```

Comment:
We should align on whether each doc article should have ##Introduction heading. @arnautov-anton @oliverlaz @szuperaz wdyt?

Reply:
It is quite messy; some chapters have an Introduction header, and some just start with the introduction...
I checked the docs of the other platforms, they omit the "## Introduction" header and they immediately start with the content. I guess we should do the same :)